src/constants/tvImage.js

```diff
@@ -0,0 +1,22 @@
+import {allTV} from "./helperTVString.js"
+
+
+
+export function tvImage (array) {
+
+    switch (array) {
+        case 0:
+        case 1:
+        case 2:
+        case 3:
+        case 4:
+        case 5:
+        case 6:
+        case 7:
+        case 8:
+            return  allTV[array].sourceImg;
+        default:
+            return "please fill in a valid input";
+
+    }
```

Comment:
Als je niet zeker weet of de index die je mee krijgt in de array zit, dan kun je ook het volgende doen:
```suggestion
        return allTV[array]?.sourceImg ?? "please fill in a valid input";
```
`?` zorgt er voor dat alleen `sourceImg` wordt gepakt als property als de entry in de array bestaat. En `??` zorgt ervoor dat de waarde die erachter staat gepakt wordt als de waarde die ervoor staat `null` of `undefined` is.